java-squid/src/main/java/org/sonar/java/ast/parser/TreeFactory.java

```diff
@@ -1651,6 +1659,7 @@ public TypeCastExpressionTreeImpl newClassCastExpression(TypeTree type, Optional
 
     List<AstNode> children = Lists.newArrayList();
     children.add((AstNode) type);
+    // TODO SONARJAVA-1139 bound should be present in castTree
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)